zh/deploy-tidb-from-kubernetes-gke.md

```diff
@@ -81,117 +75,61 @@ kubectl get nodes
 
 ## 安装 Helm
 
-[Helm](https://helm.sh/) 是一个 Kubernetes 的包管理工具，确保安装的 Helm 版本为 >= 2.11.0 && < 3.0.0 && != [2.16.4](https://github.com/helm/helm/issues/7797)。安装步骤如下：
-
-1. 参考[官方文档](https://v2.helm.sh/docs/using_helm/#installing-helm)安装 Helm 客户端
-2. 安装 Helm 服务端
-
-    在集群中应用 Helm 服务端组件 `tiller` 所需的 `RBAC` 规则，并安装 `tiller`：
+[Helm](https://helm.sh/) 是一个 Kubernetes 的包管理工具。
 
-    {{< copyable "shell-regular" >}}
+1. 安装 Helm 服务端
 
     ```shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/tiller-rbac.yaml && \
-    helm init --service-account=tiller --upgrade
+    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
     ```
 
-    通过下面命令确认 `tiller` Pod 进入 running 状态：
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    kubectl get po -n kube-system -l name=tiller
-    ```
-
-3. 通过下面的命令添加仓库：
-
-    {{< copyable "shell-regular" >}}
+2. 通过下面的命令添加仓库：
 
     ```shell
     helm repo add pingcap https://charts.pingcap.org/
     ```
 
-    添加完成后，可以使用 `helm search` 搜索 PingCAP 提供的 chart：
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    helm search pingcap -l
-    ```
-
 ## 部署 TiDB Operator
 
 TiDB Operator 使用 [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) 扩展 Kubernetes，所以要使用 TiDB Operator，必须先创建 `TidbCluster` 等各种自定义资源类型：
 
-{{< copyable "shell-regular" >}}
-
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.2/manifests/crd.yaml && \
 kubectl get crd tidbclusters.pingcap.com
 ```
 
 创建 `TidbCluster` 自定义资源类型后，接下来在 Kubernetes 集群上安装 TiDB Operator。
 
-1. 获取你要安装的 `tidb-operator` chart 中的 `values.yaml` 文件：
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    mkdir -p /home/tidb/tidb-operator && \
-    helm inspect values pingcap/tidb-operator --version=v1.1.0-rc.1 > /home/tidb/tidb-operator/values-tidb-operator.yaml
-    ```
-
-    按需修改 `values.yaml` 文件中的配置。
-
-2. 安装 TiDB Operator
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    helm install pingcap/tidb-operator --name=tidb-operator --namespace=tidb-admin --version=v1.1.0-rc.1 -f /home/tidb/tidb-operator/values-tidb-operator.yaml && \
-    kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
-    ```
-
-3. 创建 `pd-ssd` StorageClass：
-
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/persistent-disk.yaml
-    ```
+```shell
+kubectl create namespace tidb-admin
+helm install --namespace tidb-admin tidb-operator pingcap/tidb-operator --version v1.1.2
+kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
+```
```

Comment:
no need to install pd-ssd storage class anymore
https://github.com/pingcap/docs-tidb-operator/pull/527/files#r448768118